addons/hr_recruitment/data/hr_recruitment_data.xml

```diff
@@ -494,5 +494,22 @@ joining us as a ${object.job_id.name}
             <field name="name">Job Campaign</field>
     </record>
 
+    <record model="hr.recruitment.stage" id="stage_job1">
```

Comment:
Do we really want to have shared stages by default ?

Reply:
Yes